microsoft-edge/devtools-guide-chromium/whats-new/2023/07/devtools-115.md

```diff
@@ -0,0 +1,96 @@
+---
+title: What's new in DevTools (Microsoft Edge 115)
+description: "Entry 1. Entry 2. Entry 3. And more."
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 07/31/2023
+---
+# What's New in DevTools (Microsoft Edge 115)
+
+[!INCLUDE [Microsoft Edge team note for top of What's New](../../includes/edge-whats-new-note.md)]
+
+> [!TIP]
+> The **Microsoft Build 2023** conference was on May 23-25, 2023.  Learn more about new features for memory, performance, and production debugging in DevTools, as well as new capabilities for PWAs in the sidebar, WebView2, and Chat Plugins, in the following videos:
+> * [Microsoft Edge | State of web developer tools](https://www.youtube.com/watch?v=yDFmQNu3TSg&list=PL4z1-7pjJU6zJT3PBQ4mTbNg2wtX7Lt52)
+> * [Microsoft Edge | Building Progressive Web Apps for the sidebar](https://www.youtube.com/watch?v=9u8lRzRUayw&list=PL4z1-7pjJU6zJT3PBQ4mTbNg2wtX7Lt52)
+> * [Microsoft Edge | Bringing WebView2 to Microsoft Teams and beyond](https://www.youtube.com/watch?v=s3tDUvaoCP4&list=PL4z1-7pjJU6zJT3PBQ4mTbNg2wtX7Lt52)
+> * [Building Chat Plugins for Microsoft Bing and Edge](https://www.youtube.com/watch?v=Q-5M7EYjl6U&list=PL4z1-7pjJU6zJT3PBQ4mTbNg2wtX7Lt52)
+
+
+<!-- ====================================================================== -->
+## Entry 1
+
+<!-- Subtitle: . -->
+
+![todo: Alt-text 1](./devtools-115-images/template.png)
+<!-- todo: steps to re-create screenshot -->
+
+<!--
+See also:
+* []()
+-->
+
+
+<!-- ====================================================================== -->
+## Entry 2
+
+<!-- Subtitle: . -->
+
+![todo: Alt-text 2](./devtools-115-images/template.png)
+<!-- todo: steps to re-create screenshot -->
+
+<!--
+See also:
+* []()
+-->
+
+
+<!-- ====================================================================== -->
+## Entry 3
+
+<!-- Subtitle: . -->
+
+![todo: Alt-text 3](./devtools-115-images/template.png)
+<!-- todo: steps to re-create screenshot -->
+
+<!--
+See also:
+* []()
+-->
+
+
+<!-- ====================================================================== -->
+## Announcements from the Chromium project
+
+Microsoft Edge 115 also includes the following updates from the Chromium project:
+
+<!-- todo: delete some links? -->
```

Comment:
@zoherghadyali @jenna-sasson  Which links to remove?  
Rendered article section:
https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/whats-new/2023/07/devtools-115?branch=pr-en-us-2654#announcements-from-the-chromium-project

Reply:
links removed